packages/next/client/page-loader.js

```diff
@@ -70,21 +70,23 @@ export default class PageLoader {
 
       // Load the script if not asked to load yet.
       if (!this.loadingRoutes[route]) {
-        this.loadScript(route)
+        this.loadScript(route, isPreload)
         this.loadingRoutes[route] = true
       }
     })
   }
 
-  loadScript (route) {
+  loadScript (route, isPreload) {
     route = this.normalizeRoute(route)
     const scriptRoute = route === '/' ? '/index.js' : `${route}.js`
 
     const script = document.createElement('script')
     const url = `${this.assetPrefix}/_next/static/${encodeURIComponent(this.buildId)}/pages${scriptRoute}`
     script.crossOrigin = process.crossOrigin
     script.src = url
-    script.onerror = () => {
+    script.onerror = isPreload ? () => {
```

Comment:
The `loadPage` would never finish in this case right? Also the error should be caught by `.catch(` below, is it not? As far as I've seen the sentry errors for zeit.co stopped right after we shipped Next.js canary.